config/initializers/005-site_settings.rb

```diff
@@ -7,6 +7,7 @@
   RailsMultisite::ConnectionManagement.each_connection do
     begin
       SiteSetting.refresh!
+      Discourse.activate_readonly
```

Comment:
opps left over code from the previous branch. Basically, we create a thread to keep the site in readonly mode. On startup, we have to start the thread again. Otherwise, the site will fall out of readonly mode.